lib/workload/stateless/postgres_manager/function/create-pg-login-role.ts

```diff
@@ -0,0 +1,89 @@
+import { generate as pass_generator } from 'generate-password';
+import {
+  executeSqlWithLog,
+  getMicroserviceName,
+  getMicroserviceConfig,
+  getRdsMasterSecret,
+  DbAuthType,
+} from './utils';
+import {
+  SecretsManagerClient,
+  CreateSecretCommandInput,
+  CreateSecretCommand,
+} from '@aws-sdk/client-secrets-manager';
+import { Client } from 'pg';
+
+type EventType = {
+  microserviceName: string;
+};
+
+export const handler = async (event: EventType) => {
+  const microserviceConfig = getMicroserviceConfig();
+  const microserviceName = getMicroserviceName(microserviceConfig, event);
+  const pgMasterConfig = await getRdsMasterSecret();
+
+  const findConfig = microserviceConfig.find((e) => e.name == microserviceName);
+  if (findConfig?.authType != DbAuthType.USERNAME_PASSWORD) {
+    throw new Error('this microservice is not configured for username-password authentication');
+  }
+
+  const client = new Client(pgMasterConfig);
+  await client.connect();
+  console.info('connected to RDS with master credential');
+
+  // create a new role
+  console.info('creating a user-password login role');
+  const password = pass_generator({ length: 10, numbers: true });
+  const createRoleQueryTemplate = `CREATE ROLE ${microserviceName} with LOGIN ENCRYPTED PASSWORD '${password}'`;
+  await executeSqlWithLog(client, createRoleQueryTemplate);
```

Comment:
Could you change this to execute without logging, pls. Perhaps just `executeSql()` or something like that.